prepare.sh

```diff
@@ -1,7 +1,66 @@
 #!/bin/sh
 
+# These values can be modified as per the requirement.
```

Comment:
This comment doesn't make a lot of sense to an outside reader IMO. "the requirement", what requirement? Since there is not reference or link, I suggest dropping this line.

Reply:
Sure.